File: examples/13-code-examples/code_samples/create_user.py
# Python example for creating a user
import requests
import json

url = "http://localhost:8080/api/users"
payload = {
    "name": "John Doe"
}
headers = {
    "Content-Type": "application/json"
}

response = requests.post(url, data=json.dumps(payload), headers=headers)
print(response.json())

File: examples/13-code-examples/code_samples/create_user.js
// JavaScript example for creating a user
const createUser = async () => {
  const response = await fetch('http://localhost:8080/api/users', {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify({
      name: 'John Doe'
    })
  });
  
  const data = await response.json();
  console.log(data);
};

createUser();
